examples/bpk-component-map/examples.js

```diff
@@ -290,6 +351,14 @@ const WithPriceMarkersExample = () => (
   <StatefulBpkPriceMarker action={action('Price marker clicked')} />
 );
 
+const WithPriceMarkersV2Example = () => (
+  <StatefulBpkPriceMarkerV2 action={action('Price marker clicked')} style={STYLE_TYPES.default}  />
```

Comment:
I the `style` [has the default value](https://github.com/Skyscanner/backpack/pull/3007/files#diff-8e9664206cc935ad55c1f0ecacaf2b3d11004f15e55c79e71e5adc9d304cea24R68), so no need to pass here.